nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-api/src/main/java/org/apache/nifi/audit/PortAuditor.java

```diff
@@ -93,14 +91,8 @@ public Port updatePortAdvice(ProceedingJoinPoint proceedingJoinPoint, PortDTO po
         final int maxConcurrentTasks = port.getMaxConcurrentTasks();
 
         final Set<String> existingUsers = new HashSet<>();
-        final Set<String> existingGroups = new HashSet<>();
         boolean isPublicPort = false;
```

Comment:
It looks like `isPublicPort` will always be false here, as you've removed the `if (port instanceof PublicPort) {... }`. I think we need to instead set `boolean isPublicPort = port instanceof PublicPort;`